### DISCUSSION TITLE
Sort by Recently Used or Most frequently used (tap count) entry.

### DISCUSSION THREAD

**Discussion by silentdot** [ente-io/ente#488] [Enhancements]
Change the order of the different codes, have to scroll down every time to use my most used one :)

**Comment by itInner** [comment]
> Change the order of the different codes, have to scroll down every time to use my most used one :)

My solution to this problem is to add an uppercase A to the initials of the name
I think the best workaround is to add grouping functionality

**Comment by brog-io** [comment]
It'd also be nice to have multiple options. For example ordering by last used

**Comment by ThiefMaster** [comment]
Sorting by more frequently would indeed be a great feature to have and I don't think I'd move on from andOTP (or Aegis) until that's available. For people with just a handful of OTPs it's probably not a big deal but when you have 50+ then it gets really annoying if the few ones you use regularly (e.g. work and paypal and some other sites that require it for every login) aren't on top.
